File: Plugins/Aspose-Cells-Java-for-Jython/asposecells/WorkingWithWorksheets/HideUnhideWorksheet.py
from asposecells import Settings
from com.aspose.cells import Workbook


class HideUnhideWorksheet:

    def __init__(self):
        dataDir = Settings.dataDir + 'WorkingWithWorksheets/HideUnhideWorksheet/'
     
        workbook = Workbook(dataDir + "Book1.xls")
        
        #Accessing the first worksheet in the Excel file
        worksheets = workbook.getWorksheets()
        worksheet = worksheets.get(0)

        #Hiding the first worksheet of the Excel file
        worksheet.setVisible(False)

        #Saving the modified Excel file in default (that is Excel 2003) format
        workbook.save(dataDir + "output.xls")

        # Print message
        print "Worksheet 1 is now hidden, please check the output document."

if __name__ == '__main__':        
    HideUnhideWorksheet()

File: Examples/src/main/java/com/aspose/cells/examples/worksheets/HideUnhideWorksheet.java
package com.aspose.cells.examples.worksheets;

import com.aspose.cells.*;
import com.aspose.cells.examples.Utils;

public class HideUnhideWorksheet {

	public static void main(String[] args) throws Exception {

		// The path to the documents directory.
		String dataDir = Utils.getSharedDataDir(HideUnhideWorksheet.class) + "worksheets/";

		// Instantiating a Workbook object by excel file path
		Workbook workbook = new Workbook(dataDir + "book1.xls");

		// Accessing the first worksheet in the Excel file
		WorksheetCollection worksheets = workbook.getWorksheets();
		Worksheet worksheet = worksheets.get(0);

		// Hiding the first worksheet of the Excel file
		worksheet.setVisible(false);

		// Saving the modified Excel file in default (that is Excel 2003) format
		workbook.save(dataDir + "HideUnhideWorksheet_out.xls");

		// Print message
		System.out.println("Worksheet 1 is now hidden, please check the output document.");

	}
}
